app/admin/targets.rb

```diff
@@ -0,0 +1,26 @@
+ActiveAdmin.register Target do
+  permit_params :title, :lat, :lng, :radius, :topic_id
+  index do
+    selectable_column
+    id_column
+    column :title
+    column :radius, min: 0
+    column :lat
+    column :lng
+    column :topic
+    column :user
+    actions
+  end
+  filter :title
+  filter :radius
+  filter :lat
+  filter :lng
```

Comment:
keep in mind that applying filters by non-indexed columns can cause low performance problems. Consider adding indexes in the db or removing unnecessary filters if its posible

Reply:
Let me know if like this its ok. Thanks!